### PR TITLE
Add "You-" as a prefix if it's you (:

### PR DESCRIPTION
**Pull request by MaitreRouge**
**Please describe the changes this PR makes and why it should be merged**
Soo this pull request is very important, fix a major flaw in the potato farm and I think our production will increse after the merge (maybe +47%)
Nah I'm just kidding, I'm just lonely, I feel bad and I rly want a hug

Ow, beside my mental health being rly bad, this PR add a " - YOU" prefix on the levels page if it's you (:

**In this PR:**
- Code changes were not tested
- Strings were added/removed (new/deleted string keys)

### CODE REVIEW

**Review by ImRodry — DISMISSED** [review]
You should add a string just with the text "YOU" and then add it to the right side of the member tag, instead of the left side.
Also you can remove the comment, it will work :)

**Comment by MaitreRouge** [comment]
I applied ethe changes I was asked for

File: src/commands/Info/levels.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ function fetchPage(page: number, pages: DbUser[][], getString: (path: string, va
         if (pages[page][i].levels) {
             const totalXp = pages[page][i].levels!.totalXp
             pageEmbed.addField(getString("level", { rank: (i + 1) + (page * 24), level: pages[page][i].levels!.level, xp: totalXp > 1000 ? `${(totalXp / 1000).toFixed(2)}${getString("thousand")}` : totalXp }), `<@!${pages[page][i].id}>`, true)
-        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}>`, true)
+        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}> ${pages[page][i].id === interaction.user.id ? getString("youIndicator") : ""}`, true)
```
Comment (ImRodry) [inline src/commands/Info/levels.ts:110]:
```suggestion
        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}> ${pages[page][i].id === interaction.user.id ? ` - ${getString("youIndicator")}` : ""}`, true)
```

**Comment by MaitreRouge** [comment]
Think I've done everything needed, gtg watch smth on netflix now :D

**Comment by ImRodry** [comment]
@Mergifyio refresh

File: src/commands/Info/levels.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ function fetchPage(page: number, pages: DbUser[][], getString: (path: string, va
         if (pages[page][i].levels) {
             const totalXp = pages[page][i].levels!.totalXp
             pageEmbed.addField(getString("level", { rank: (i + 1) + (page * 24), level: pages[page][i].levels!.level, xp: totalXp > 1000 ? `${(totalXp / 1000).toFixed(2)}${getString("thousand")}` : totalXp }), `<@!${pages[page][i].id}>`, true)
```
Comment (ImRodry) [inline src/commands/Info/levels.ts:109]: Forgot this line ;)
```suggestion
            pageEmbed.addField(getString("level", { rank: (i + 1) + (page * 24), level: pages[page][i].levels!.level, xp: totalXp > 1000 ? `${(totalXp / 1000).toFixed(2)}${getString("thousand")}` : totalXp }), `<@!${pages[page][i].id}>${pages[page][i].id === interaction.user.id ? ` - ${getString("youIndicator")}` : ""}`, true)
```

**Review by ImRodry — CHANGES_REQUESTED** [review]
Ignore whatever this is

File: src/commands/Info/levels.ts (hunk 2 above)
Comment (ImRodry) [inline src/commands/Info/levels.ts:109]: Forgot this line
```suggestion
            pageEmbed.addField(getString("level", { rank: (i + 1) + (page * 24), level: pages[page][i].levels!.level, xp: totalXp > 1000 ? `${(totalXp / 1000).toFixed(2)}${getString("thousand")}` : totalXp }), `<@!${pages[page][i].id}>${pages[page][i].id === interaction.user.id ? ` - **${getString("youIndicator")}**` : ""}`, true)
```

File: src/commands/Info/levels.ts [hunk 3]
```diff
@@ -107,7 +107,7 @@ function fetchPage(page: number, pages: DbUser[][], getString: (path: string, va
...
-        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}>`, true)
+        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}>${pages[page][i].id === interaction.user.id ? ` - ${getString("youIndicator")}` : ""}`, true)
```
Comment (ImRodry) [inline src/commands/Info/levels.ts:110]: Making this bold looks better
```suggestion
        } else pageEmbed.addField(getString("unranked", { rank: (i + 1) + (page * 24) }), `<@!${pages[page][i].id}>${pages[page][i].id === interaction.user.id ? ` - **${getString("youIndicator")}**` : ""}`, true)
```

**Review by ImRodry — CHANGES_REQUESTED** [review]
Now for real

**Review by ImRodry — APPROVED** [review]
Looks good, thank you for your help!

**Pull request merged**